pontoon/base/models.py

```diff
@@ -2287,6 +2309,7 @@ class Entity(DirtyFieldsMixin, models.Model):
     order = models.PositiveIntegerField(default=0)
     source = JSONField(blank=True, default=list)  # List of paths to source code files
     obsolete = models.BooleanField(default=False)
+    word_count = models.PositiveIntegerField(default=0)
```

Comment:
I've added word_count property, but to make it work I need to run migration, am I right?

Reply:
Correct:
https://docs.djangoproject.com/en/3.0/topics/migrations/#workflow